### PR TITLE
Readme updated

### PR DESCRIPTION
**Pull request by jujusudhur**
**Fixes issue:** #2848 

**Changes:**
Added an Introduction about Bubble sort, Overview, Complexity and Further reading.

### CODE REVIEW

File: code/sorting/bubble_sort/README.md [hunk 1]
```diff
@@ -1,11 +1,50 @@
...
+**Bubble Sort** is the simplest sorting algorithm that works by repeatedly swapping the adjacent elements if they are in wrong order. In computer graphics it is popular for its capability to detect a very small error (like swap of just two elements) in almost-sorted arrays and fix it with just linear complexity (2n). For example, it is used in a polygon filling algorithm, where bounding lines are sorted by their x coordinate at a specific scan line (a line parallel to x axis) and with incrementing y their order changes (two elements are swapped) only at intersections of two lines.
 
-## Procedure
-1. Run through the list.
-2. Compare each adjacent pair of element
-3. Swap elements if not in correct order
-4. Repeat until no swap is needed
+## Overview
+**First Pass:**
+<br>
```
Comment (arnavb) [inline code/sorting/bubble_sort/README.md]: These line breaks are somewhat unnecessary.

**Comment by arnavb** [comment]
No, no, no! You misunderstood. I meant that you should add newlines _instead_ of \<brs>. But you can keep it as it is if you want! 😆

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by jujusudhur** [comment]
Thank you

**Pull request merged**